test/unit/signup/account_manager_test.rb

```diff
@@ -127,7 +127,6 @@ class ProviderAccountManagerTest < Signup::AccountManagerTest
 
       test 'first service has a private endpoint in order to be functional for non-APIAP accounts' do
         Account.any_instance.stubs(:provider_can_use?).returns(false)
-        Account.any_instance.stubs(:provider_can_use?).with(:api_as_product).returns(false)
```

Comment:
The changes in this file don't belong to this PR. And reading the title of the tests, these changes don't match what the title says either 😞

Reply:
Proposal done in https://github.com/3scale/porta/pull/2294 so it is in a logic unit (and I added tests for related untested stuff before actually changing the code)